env_origin.yaml

```diff
@@ -19,4 +19,5 @@ parameters:
   openshift_ansible_git_rev: master
 
 resource_registry:
-    OOShift::LoadBalancer: loadbalancer_neutron.yaml
+  OOShift::LoadBalancer: loadbalancer_neutron.yaml
+  OOShift::ContainerPort: sdn_openshift_sdn.yaml
```

Comment:
Hm, now I see loadbalance ris missing there too, so ignore this - we can add them in a separate patch.